nginx/common.sls

```diff
@@ -22,6 +22,11 @@
 /usr/share/nginx:
   file:
     - directory
+    
+apache2:
```

Comment:
This doesn't feel like something the formula should handle. The end user should be able to deal with this themselves, by including this state in their own salt configuration.

Forcing this on formula users removes the ability to have apache and nginx on the same machine (not sure why someone would ever want that, but I don't see a good reason to break it).